src/test/fuzz/asmap_direct.cpp

```diff
@@ -6,27 +6,27 @@
 #include <test/fuzz/fuzz.h>
 
 #include <cstdint>
+#include <optional>
 #include <vector>
 
 #include <assert.h>
 
 void test_one_input(const std::vector<uint8_t>& buffer)
 {
     // Encoding: [asmap using 1 bit / byte] 0xFF [addr using 1 bit / byte]
-    bool have_sep = false;
-    size_t sep_pos;
+    std::optional<size_t> sep_pos_opt;
     for (size_t pos = 0; pos < buffer.size(); ++pos) {
         uint8_t x = buffer[pos];
         if ((x & 0xFE) == 0) continue;
         if (x == 0xFF) {
-            if (have_sep) return;
-            have_sep = true;
-            sep_pos = pos;
+            if (sep_pos_opt) return;
+            sep_pos_opt = pos;
         } else {
             return;
         }
     }
-    if (!have_sep) return; // Needs exactly 1 separator
+    const size_t sep_pos{sep_pos_opt.value()};
+    if (!sep_pos) return; // Needs exactly 1 separator
```

Comment:
The two lines are swapped, no?
```suggestion
    if (!sep_pos_opt) return; // Needs exactly 1 separator
```

Reply:
Yeah, these errors happen when you have no working compiler on your machine. Can't compile fuzzing variants anymore (*Catalina is not an OS but Malware*). It's time for Linux VM.

Sorry!